scripts/variantstore/wdl/GvsAssignIds.wdl

```diff
@@ -9,7 +9,7 @@ workflow GvsAssignIds {
     String dataset_name
     String project_id
 
-    Array[String] external_sample_names
+    File bulk_import_tsv
```

Comment:
I was unaware that there was a desire to run any of the subworkflows called by `GvsBulkIngest` individually during dev. We can certainly make changes to support this but it would be helpful to capture the requirements first.